test/UtilityRepository-test.js

```diff
@@ -20,7 +20,7 @@ beforeEach(() => {
       { userID: 45, date: "2019/08/24", hoursSlept: 8.3, sleepQuality: 1.5 }
     ];
 
-  utilityRepo = new UtilityRepository([],dataSleep,[],[])
+  utilityRepo = new UtilityRepository([], dataSleep, dataActivity, [])
```

Comment:
Was the activity not working without this?  We don't use dataActivity in the utilityRepository test suite.

Reply:
YES! I totally forgot to take that out. Sorry! I'll take care of that.